### PR TITLE
rustup: 2017-10-29 -> 1.11.0

### PR DESCRIPTION
**Pull request by dywedir**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

cc @Mic92

### CODE REVIEW

**Comment by dywedir** [comment]
@GrahamcOfBorg buld rustup

**Comment by dywedir** [comment]
@GrahamcOfBorg build rustup

**Comment by Mic92** [comment]
Thanks!

File: pkgs/development/tools/rust/rustup/default.nix [hunk 1]
```diff
@@ -35,23 +35,23 @@ rustPlatform.buildRustPackage rec {
   postInstall = ''
     pushd $out/bin
     mv rustup-init rustup
-    for link in cargo rustc rustdoc rust-gdb rust-lldb; do
+    for link in cargo rustc rustdoc rust-gdb rust-lldb rls rustfmt cargo-fmt; do
```
Comment (Mic92) [inline pkgs/development/tools/rust/rustup/default.nix]: cc @steveeJ

**Pull request merged**